ecommerce-ios/Extensions/Collection/Array+Safe.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Array+Safe.swift
+//  ecommerce-ios
+//
+//  Created by Nguyen M. Tam on 13/05/2021.
+//
+
+import Foundation
+
+extension Array {
+
+    subscript (safe index: Index) -> Element? {
```

Comment:
maybe we don't need the extra space here?
```suggestion
    subscript(safe index: Index) -> Element? {
```